.github/workflows/build.yml

```diff
@@ -0,0 +1,124 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  upload-src:
+    name: Upload the source
+    runs-on: ubuntu-latest
+    steps:
+      - name: prep ubuntu
+        run: |
+          sudo apt-get update
+          DEBIAN_FRONTEND=noninteractive sudo apt-get install -yq \
+            ${{ matrix.pkgs }} cmake git golang libbrotli-dev libfmt-dev \
+            libgtest-dev liblz4-dev libpcre2-dev libprotobuf-dev libunwind-dev \
+            libusb-1.0-0-dev libzstd-dev make pkg-config
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: git config
+        run: |
+          git config --global user.email "you@example.com"
+          git config --global user.name "Your Name"
+      - name: build & install
+        run: |
+          mkdir build && cd build
+          cmake ..
+          make package_source
+      - name: upload package_source
+        uses: actions/upload-artifact@v2
+        with:
+          name: package_source
+          path: ${{ github.workspace }}/build/android-tools-*.tar.xz
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: "opensuse/leap:15.3"
+            pkgs: "clang gcc10 gcc10-c++"
+            env1: "CC=clang CXX=clang++"
+          - os: "opensuse/tumbleweed"
+            pkgs: "gcc gcc-c++"
+            env1: "CFLAGS='-Wno-error=array-parameter -Wno-error=array-bounds -Wno-error=stringop-overflow'"
```

Comment:
These warnings are disabled for gcc 11 I suppose? Shouldn't this have been fixed with #34 or is this a different issue I am not aware of?